modules/hcp-dc-mgmt-cluster.adoc

```diff
@@ -133,7 +133,7 @@ vmrules:
       mac: aa:aa:aa:aa:10:03
 ----
 +
-<1> Replace `4.x.y` with the supported {product-title} version you want to use.
+<1> Replace `<4.x.y>` with the supported {product-title} version you want to use.
```

Comment:
In the YAML, be sure to place angle brackets around any instances of `4.x.y` to match the text in the annotation.